client/src/includes/tabs.js

```diff
@@ -294,7 +294,7 @@ class Tabs {
   }
 
   /**
-   * Set url to have tab an tab hash at the end
+   * Set url to have tab a tab hash at the end
```

Comment:
This doesn't really make any more sense than the original wording. I wondered if it was meant to be "tab and tab hash", but I can't see anything in the actual code to justify that. So, I'd go with:
```suggestion
   * Set url to have a tab hash at the end
```